helm/templates/deployment-controller.yaml

```diff
@@ -81,6 +81,8 @@ spec:
           value: "true"
         - name: __SUPERBLOCKS_WORKER_LOCAL_ENABLED
           value: "true"
+        - name: SUPERBLOCKS_AGENT_DATA_DOMAIN
+          value: {{ default "app.superblocks.com" .Values.superblocks.agentDataDomain | quote }}
```

Comment:
Think we might wanna add a strict check to ensure it is one of app.superblocks.com or eu.superblocks.com